README.md

```diff
@@ -6,6 +6,16 @@
 This is an [EditorConfig][] plugin for Vim. This plugin can be found on both
 [GitHub][] and [Vim online][].
 
+## Bundled versions
+
+Depending on which version of Vim or Neovim you are using, you might not need to
+install this plugin at all:
+
+* Vim 9.0.1799 and above comes bundled with [a version of this plugin][].
+* Neovim 0.9 and above comes with [its own Lua-based implementation][].
+
+Both support the usual [properties][], including `max_line_length`.
```

Comment:
Is this paragraph needed? I feel this one causes more confusion.

Reply:
It's there to assure people that the bundled implementations are not a "light" version that's missing any features. For me personally this was one of the main questions that were on my mind when deciding whether to use the bundled version or install this plugin. Therefore I think that it makes sense to have at least something like this sentence. But if you have any suggestions on how to rephrase it, go ahead! :)

Reply:
OK, no worry! I'll make changes